pkg/deploy/extra_images.go

```diff
@@ -0,0 +1,8 @@
+// This file is generated, and contains the latest versions of certain properties from che.properties
+package deploy
+
+const (
+	cheWorkspacePluginBrokerInitImage    = "eclipse/che-init-plugin-broker:v0.21"
```

Comment:
I would prefer if these contained the registry in the URL (docker.io) as a reminder of things we need to migrate to quay.

Also don't love that jwtproxy does SHA releases, not numbered ones (7.3.0). But those are out of scope for this issue. :)